dbms/src/IO/WriteBufferFromS3.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <functional>
```

Comment:
Way too many headers in .h file. Will move to .cpp.

Reply:
6fb9565091264bd0abc5aa43b85eac0db309c682